### DISCUSSION TITLE
Add OSTree Support

### DISCUSSION THREAD

**Discussion by mystic-githuber** [mullvad/mullvadvpn-app#8913] [Feature requests & Ideas]
Dear Devs,

Fedora's Immutable spins are becoming popular. It would be great if you could provide OSTree support similar to iVPN.

[https://www.ivpn.net/knowledgebase/linux/fedora-silverblue/](url)

**Comment by CaptainMuskrat** [comment]
As a Fedora Silverblue user, I wholeheartedly agree. It's why I also suggested a Flatpak or even an Appimage release too.
